trnsysGUI/mainWindow.py

```diff
@@ -174,6 +174,17 @@ def askForSaveAndLoadSettings() -> None:
         self.editMenu.addAction(toggleSnapAction)
         self.editMenu.addAction(toggleAlignModeAction)
 
+        from trnsysGUI.menus.hydraulicModesMenu import hydraulicModes as hm
```

Comment:
These circular import only arise due to type checking.

if typing.TYPE_CHECKING:
 import